src/components/Deletemodal.tsx

```diff
@@ -0,0 +1,89 @@
+import { motion } from 'framer-motion';
+import { LiaTimesSolid } from 'react-icons/lia';
+import { RiDeleteBinFill } from 'react-icons/ri';
+import { useAppDispatch } from '../redux/hooks/hooks';
+import useToast from '../hooks/useToast';
+import { createWish } from '../redux/features/createWishesSlice';
+import { fetchWishes } from '../redux/features/getUserwishes';
+import { DynamicData } from '../@types/DynamicData';
+
+interface DeleteModalType {
+	setDeleteModal: (D: string) => void;
+	productId: string;
+}
+function DeleteModal({ setDeleteModal, productId }: DeleteModalType) {
+	const dispatch = useAppDispatch();
+	const { showErrorMessage, showSuccessMessage } = useToast();
+	const handleAddRemoveWish = async (productId: string) => {
+		try {
+			await dispatch(createWish({ productId })).unwrap();
+			showSuccessMessage('removed to wish successfully');
```

Comment:
for this message, you should display the one from backend side not hard coded message